src/containers/forms/LogActivityForm.jsx

```diff
@@ -225,7 +225,7 @@ class LogActivityForm extends Component {
         <div className='titleForm'>{formTitle}</div>
         <DateField
           handleChange={this.handleChange}
-          value={this.state.date}
+          value={this.state.activityDate}
```

Comment:
@puritybirir Destructure `activityDate` from `this.state` for consistency.

Reply:
On it

Reply:
Done